src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * @param string $sku
+     * @param int    $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
```

Comment:
I was thinking about this a bit - yes, commands, queries are stupid DTOs and should contain only simple types.
But they can still require VOs as arguments and cast them internally, just to ensure correctness and so validation rules are in VOs and not in each command again and again and again.